faststream/confluent/prometheus/middleware.py

```diff
@@ -0,0 +1,25 @@
+from typing import TYPE_CHECKING, List, Optional
+
+from faststream.confluent.prometheus.provider import settings_provider_factory
+from faststream.prometheus.middleware import BasePrometheusMiddleware
+
+if TYPE_CHECKING:
+    from prometheus_client import CollectorRegistry
+
+
+class KafkaPrometheusMiddleware(BasePrometheusMiddleware):
+    def __init__(
+        self,
+        *,
+        registry: "CollectorRegistry",
+        app_name: str = "faststream",
+        metrics_prefix: str = "faststream",
+        received_messages_size_buckets: Optional[List[float]] = None,
```

Comment:
Why is it `Optional[List[float]] = None`? Could it be `Sequence/Iterable[float] = ()`?

Reply:
I found it necessary not to write `Sequence/Iterable[float] = ()`, but simplified from List to Sequence